api/routes/favourites.js

```diff
@@ -14,12 +14,13 @@ router.get('/', function(req, res, next) {
         }
         console.log('Connected...');
         const collection = client.db("travelr").collection("favourites");
-        collection.find({}).toArray((err, result) => {
-            if (err) console.log(err)
-            else {
-                res.send(result);
-            }
-        })
+        // collection.find({}).toArray((err, result) => {
+        //     if (err) console.log(err)
+        //     else {
+        //         return res.send(result);
+        //     }
+        // })
+        let response = collection.find({});
```

Comment:
This change has nothing to do with the Youtube querying. I made this change b/c the original code was making infinite calls to the db on a loop.